### DISCUSSION TITLE
All the different models and what they do.

### DISCUSSION THREAD

**Discussion by githubUser01946** [Mikubill/sd-webui-controlnet#439] [General]
Is there a list that we can see that explains the different models and what they do, for example we know that Depth will make depth images or something but which model is better for faces/whats better for buildings, that kind of thing.

**Comment by ljleb** [comment]
checkout the readme on the original repo: https://github.com/lllyasviel/ControlNet#production-ready-pretrained-models
not much explanations by word but you can guess what models do pretty easily with the images

**Reply by githubUser01946** [reply]
Rightio. Thanks
